examples/topics/stats/density.py

```diff
@@ -0,0 +1,61 @@
+''' A `multiple kernel density estimation`_ plot of the "cows" data using
+the `sklearn.neighbors.KernelDensity`_ function and `Bokeh` varea glyph
+
+.. bokeh-example-metadata::
+    :sampledata: cows
+    :apis: bokeh.plotting.figure.varea
+    :refs: :ref:`ug_topics_stats_density`
+    :keywords: density, varea
+
+.. _kernel density estimation: https://en.wikipedia.org/wiki/Kernel_density_estimation
+.. _sklearn.neighbors.KernelDensity: https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KernelDensity.html
+
+'''
+import numpy as np
+from sklearn.neighbors import KernelDensity
+
+from bokeh.models import ColumnDataSource, Label, PrintfTickFormatter
+from bokeh.palettes import Dark2
+from bokeh.plotting import figure, show
+from bokeh.sampledata.cows import data as df
+
+grouped = df.groupby('breed')
+colors = Dark2[5]
+x = np.linspace(2, 8, 1000)
+source = ColumnDataSource(dict(x=x))
+
+p = figure(title="Multiple density estimates", height=300, x_range=(2.5, 7.5), x_axis_label="butterfat contents", y_axis_label="density")
+
+for i, (breed, group_df) in enumerate(grouped):
+    data = group_df['butterfat'].values
+    kde = KernelDensity(kernel="gaussian", bandwidth=0.2).fit(data[:, np.newaxis])
+    log_dens = kde.score_samples(x[:, np.newaxis])
```

Comment:
we have enough space to avoid abbreviations
```suggestion
    log_density = kde.score_samples(x[:, np.newaxis])
```